### PR TITLE
Image input

### PR DESCRIPTION
**Pull request by Prathyusha-96**

### CODE REVIEW

File: components/image/image.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 1rem;
+    margin-bottom: 1.5rem;
+    flex-wrap: wrap;
+}
+div {
+    display: block;
+}
```
Comment (shaik-saad) [inline components/image/image.css]: DIv is  by default block

**Review by Abhijeet43 — CHANGES_REQUESTED** [review]
Hey @Prathyusharam your input component should have validation and error inputs as well. Please add them.

File: components/image/image.css [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+.rsp-img {
+    width: 100%;
+    height: autoo;
```
Comment (shaik-saad) [inline components/image/image.css]: the height value has a typo

**Review by shaik-saad — APPROVED** [review]
Please add description from next pr as has Atul mentioned.

**Review by Abhijeet43 — APPROVED** [review]
Code Looks Good now!!!!!!!

**Pull request merged**